controllers/tutorial.controller.js

```diff
@@ -71,14 +77,18 @@ const deleteTutorial = async (req, res) => {
 
     if (tutorial === null) {
       res.status(404).send({ message: "Tutorial not found" });
+      logger.tutorialLogger.log('error', 'Tutorial not found');
```

Comment:
We only need to log 500 (any 5xx errors) errors. No need to log 400, 404, or any other 4xx errors.

CC: @jaimin45 @ashishkava